Travel Project v1/Event/EventLogic.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Event
+{
+
+    public delegate void HandleEvent(double d);
```

Comment:
please consider delagate declaration in separate file

Reply:
It's pretty convenient to locate delegate in the same file with event, sry (: